packages/appstore/webpack.config.js

```diff
@@ -153,7 +153,7 @@ module.exports = function (env) {
                 },
                 {
                     test: /\.svg$/,
-                    issuer: /\/packages\/wallets\/.*(\/)?.*.tsx/,
+                    issuer: /\/packages\/wallets\/.*(\/)?.*.(ts|tsx)/,
```

Comment:
This allows the SVGs imported in `currencies.ts` using `import(...)` to be loaded as SVGR component